test/evergreen/build_windows.ps1

```diff
@@ -1,9 +1,29 @@
 param (
     [bool]$configure = $false,
     [bool]$build = $false,
-    [string]$vcvars_bat = "C:\Program Files\Microsoft Visual Studio\2022\Professional\VC\Auxiliary\Build\vcvars64.bat"
+    [string]$vcvars_bat = $null
 )
 
+# If not provided, try to auto-detect
+if (-not $vcvars_bat) {
+    $vcvars_paths = @(
```

Comment:
Do we need to care about other VS versions? WDYTA searching for this bat file using:
```
Get-ChildItem -Path "C:\Program Files (x86)\Microsoft Visual Studio" -Filter "vcvars64.bat" -Recurse -ErrorAction SilentlyContinue
```

Or using `vswhere` to locate lates VS version path that can be faster than scanning the whole "Microsoft Visual Studio" directory:

```
"%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe" -products * -latest -property installationPath
```

Reply:
It's a good idea to use `vswhere`. I'll just narrow it down to find a version that's not older than 2022.